cli/tauri-bundler/src/sign.rs

```diff
@@ -0,0 +1,156 @@
+extern crate minisign;
```

Comment:
Sure! Before I proceed I need to validate few things.

Because right now how it works;

Bundler generates custom updater artifacts as you can see here, and then sign the `zip` or `tar.gz`
https://github.com/tauri-apps/tauri/pull/643/files#diff-4f1d416019fc14a2e07e8f0d6c99717ebe1ec898337945bdad8b13535acb5c42R45

If we want to have the sign in the core, we need to be able to know what's the updater artifact path, right now, we run the build without reading paths:
https://github.com/tauri-apps/tauri/blob/feature/new_updater/cli/core/src/build.rs#L162

I can read them and sign all `zip` and `tar.gz` as it _should be_ the updater artifacts.

As you probably already noticed, the signer is required in the core as well;
https://github.com/tauri-apps/tauri/blob/feature/new_updater/cli/core/src/sign.rs#L64

Should I modify the return of `bundle_project` to be able to pass a type with the PathBuf so I know at 100% its an updater artifact?

Reply:
That's a good idea :)

Reply:
here we go!

![Screen Shot 2021-03-24 at 12 26 15 PM](https://user-images.githubusercontent.com/22237916/112346256-28123200-8c9c-11eb-9fc2-d64cc95b602f.png)